oeis.py

```diff
@@ -724,6 +724,58 @@ def A002182() -> Iterable[int]:
             yield n
 
 
+@oeis.from_function()
+def A008586(n: int) -> int:
+    """Multiples of 4"""
+    return n * 4
+
+
+@oeis.from_function()
+def A008590(n: int) -> int:
+    """Multiples of 8"""
+    return n * 8
+
+
+@oeis.from_function()
+def A008591(n: int) -> int:
+    """Multiples of 9"""
+    return n * 9
+
+
+@oeis.from_function()
+def A000578(n: int) -> int:
+    """The cubes"""
+    return n ** 3
+
+
+@oeis.from_function()
+def A000583(n: int) -> int:
+    """Fourth powers"""
+    return n ** 4
+
+
+@oeis.from_function()
+def A000584(n: int) -> int:
+    """Fifth powers"""
+    return n ** 5
+
+
+@oeis.from_function()
+def A000196(n: int) -> int:
+    """Integer part of square root of n"""
+    return int(n ** 0.5)
```

Comment:
I think `math.isqrt` would do a better job here, but it only exists since 3.8, and oeis is compatible with 3.7.

I don't want to drop 3.7 compatibility today, so you could add something like:

```python
if sys.version_info > (3, 8):
    ...
```

or:

```python
try:
    return math.isqrt(n)
except AttributeError:
    return int(n ** 0.5)
```

?

The idea of `isqrt` is to avoid the imprecision of using floats (which only provides like 13 significative digits).
 
TBH nobody should spot the difference, 13 digits is already big, and we need to get a little further to spot the difference after flooring it.

But we can see differences:

```python
>>> i = 10**16-1  # A bit more than 13 digits
>>> int(i ** .5)
100000000  # "Obviously" wrong: it should be a tad less than that, as it's exactly `sqrt(i + 1)`. 
>>> math.isqrt(i)
99999999  # isqrt get it right!
```

To play with bigger numbers, like 2 million nines, `math.isqrt` is quite robust:

```python
>>> i = 10 ** 2_000_000 - 1
>>> math.isqrt(i) ** 2 < i < (math.isqrt(i) + 1) ** 2
True
```